Readme.md

```diff
@@ -93,7 +93,7 @@ nk_end(&ctx);
 
 ## Bindings
 There are a number of nuklear bindings for different languges created by other authors.
-I cannot atest for their quality since I am not necessarily proficient in either of these
+I cannot atest for their quality since I am not necessarily proficient in any of these
```

Comment:
@sglass68: Thanks! There's another typo here, "atest" should be "attest"

Reply:
yes